Libraries/test/Amazon.Lambda.Annotations.SourceGenerators.Tests/SourceGeneratorTests.cs

```diff
@@ -83,6 +90,12 @@ public async Task SimpleCalculator()
                     }
                 }
             }.RunAsync();
+
+            var expectedTemplateContent = File.ReadAllText(Path.Combine("Snapshots", "ServerlessTemplates", "simpleCalculator.template"));
+            var actualTemplateContent = File.ReadAllText(Path.Combine("TestServerlessApp", "serverless.template"));
+            expectedTemplateContent = expectedTemplateContent.Replace("\r\n", "\n");
+            actualTemplateContent = actualTemplateContent.Replace("\r\n", "\n");
```

Comment:
do not modify the actual template. Only modify the expected template to make it compatible with the environment where the test are being run.

Reply:
Fixed